README.md

```diff
@@ -1,67 +1,180 @@
-# Polywrap python client
+![Public Release Announcement](https://user-images.githubusercontent.com/5522128/177473887-2689cf25-7937-4620-8ca5-17620729a65d.png)
+
+# Polywrap Python Client
+
+## [Polywrap](https://polywrap.io) is a developer tool that enables easy integration of Web3 protocols into any application. It makes it possible for applications on any platform, written in any language, to read and write data to Web3 protocols.
+
+
+# Working Features
+
+This MVP Python client enables the execution of **[WebAssembly](https://en.wikipedia.org/wiki/WebAssembly) Polywrappers** *(or just "wrappers")* on a python environment. It's built following the functionality of the [JavaScript Polywrap Client](https://github.com/polywrap/toolchain), which is currently more robust out and battle tested, as it has additional capabilities than this MVP.
+
+In the future, the Polywrap DAO will continue improving this Python capabilities to reach feature parity with the JS stack, including the possibiity of creating WASM wrappers with Python code. 
+
+Here you can see which features have been implemented on each language, and make the decision of which one to use for your project.
+
+| Feature | [Python](https://github.com/polywrap/python-client) | [Javascript](https://github.com/polywrap/toolchain) | 
+| -- | -- | -- |
+| Invoke wrappers | yes | yes |
+| polywrap-asyncify | replaced with wasmtime | yes |
+| polywrap-uri-resolvers | legacy | refactored |
+| polywrap-msgpack| yes, tested 100% | yes |
+| Subinvoke | wip | yes |
+| Wrap Manifest | WIP | yes |
+| Client Config Builder Package | wip | yes |
+| Interfaces | pending | yes |
+| e2e Tests | tbd | no |
+| Creating Plugins | tbd | yes |
+| Creating Python Wrappers | in a future version | yes |
+> TODO: Update table above according to test harness and maybe mention other wip clients (go, rust)
```

Comment:
Features and package are different things and we should only have feature list here.

- Invoke
- Subinvoke
- Interfaces
- Env
- Config builder
- Plugin
- Manifest
- Uri Resolution